assets/defold-event.json

```diff
@@ -17,6 +17,73 @@
     "HTML5"
   ],
   "project_url": "https://github.com/Insality/defold-event",
+  "release_tags": [
```

Comment:
not all the repos have releases, even if repo uses releases, it's not necessary true that each tag has a release, so @Insality asked to add tags as well as releases into json